flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1945,8 +1938,4 @@ Map<String, String> generateApplicationMasterEnv(
         Utils.setupYarnClassPath(this.yarnConfiguration, env);
         return env;
     }
-
-    private String getDisplayMemory(long memoryMB) {
```

Comment:
Even if we use a long value, converting to byte here will still overflow, right?

Reply:
It seems that some test case related to this is failed.

Reply:
Sorry, it was my mistake, I lost part of the code when I rebase, I re-updated the code.